### ISSUE TITLE
SPIKE: Address Solr file limit warning

### ISSUE THREAD

**Issue by maxkadel** [yalelibrary/YUL-DC#537]
On deploying a new Solr service, we get this warning. Should probably be addressed in the Solr docker image.

```
*** [WARN] *** Your open file limit is currently 1024.
It should be set to 65000 to avoid operational disruption.
If you no longer wish to see this warning, set SOLR_ULIMIT_CHECKS to false in your profile or solr.in.sh
```

If we can, set to 65,000.  If not, suppress error.

**Comment by dylansalay** [comment]
PR: https://github.com/yalelibrary/yul-dc-camerata/pull/284

Before:
![101820080-f43f4200-3ada-11eb-9b12-1a05701cabcd.png](https://images.zenhubusercontent.com/5eac63602bc3d4b76b1f451c/d4f8a6a2-71c7-443c-8317-0e9c5ac87faf)

After:
![101820231-25b80d80-3adb-11eb-9b48-03b98309342a.png](https://images.zenhubusercontent.com/5eac63602bc3d4b76b1f451c/60372b81-453e-461f-8724-5750149e60fe)

**Comment by fnibbit** [comment]
![image.png](https://images.zenhubusercontent.com/5e4c418d8f780f6e39f8f8ce/439cf789-ac13-410d-ba03-75396d5796b9)  

with solr v1.0.5 and camerata v4.0.0

**Comment by fnibbit** [comment]
although ![image.png](https://images.zenhubusercontent.com/5e4c418d8f780f6e39f8f8ce/0ec1019b-ffa2-44d4-b8a3-24a0808df4a4) from the task definition for that same solr deploy.

**Comment by DraxIndustries79** [comment]
Recent changes to solr image required us to rethink the approach (Dylan adding notes)

**Comment by dylansalay** [comment]
Notes:

When we were able to deploy successfully without seeing the warning, we deployed solr:v1.0.2, which was running as user=solr.

One more recent versions, user was changed to root (https://github.com/yalelibrary/yul-dc-solr/pull/16/files). 

The ulimit is set at a user level so when we spin up the container as root, the solr ulimit is **not** being set by our configuration in the solr-compose.yml template file, the root ulimit is.

Screenshot below shows a running solr:v1.0.4 (user=root) with the root open file limit set to 65000, as configured in the solr-compose.yml template file.
Changing the user to solr we can see that the open file limit is 1024. We can set this manually within the container by running `ulimit -n 65000`. How can we set this before/during container spin up?

![Screen Shot 2020-12-16 at 6.27.29 PM.png](https://images.zenhubusercontent.com/5eac63602bc3d4b76b1f451c/62899bbe-2ce1-40bf-a977-4f35e5044f99)

Next steps:
Determine how to set the solr ulimit -n (open file limit) to 65000  
OR
Suppress the error by setting the SOLR_ULIMIT_CHECKS to false in your profile or solr.in.sh
